plugins/modules/ecs_ecr.py

```diff
@@ -85,6 +85,24 @@
         default: false
         type: bool
         version_added: 1.3.0
+    encryption_configuration:
+        description:
+            - The encryption configuration for the repository.
+        required: false
+        suboptions:
+            encryptionType:
```

Comment:
It's generally preferred to use snake_case rather than camelCase for option and suboption names.  I know that this is being directly passed to the API, however we try to avoid mixing things up unless absolutely necessary (for example we don't mess with IAM policies where case can be important)